### ISSUE TITLE
Simplify the ui-base-theme

### ISSUE THREAD

**Issue by HeroicEric** [prototypal-io/untitled-ui#106]
There are some things in the base theme that I think we eventually want to move out. They initially a good way to test out some of our concepts but I don't think they belong in the base theme and should maybe move to a separate theme.

This issue is a place to keep track of them as I run into them.

Things to determine if they belong in the base theme:
- [ ] `{{ui-panel kind="ios"}}`
- [ ] `{{ui-button kind="material"}}`

**Comment by jacojoubert** [comment]
Yeah, the mobile things were just for testing. Same with the `ui-ripple` component. It needs to get deleted.
